wled00/wled.cpp

```diff
@@ -864,3 +865,12 @@ void WLED::handleStatusLED()
   }
   #endif
 }
+
+void WLED::disableWiFi() {
+  wifiDisabled = true;
+  WiFi.disconnect(true);
+}
+
+void WLED::enableWiFi() {
+  wifiDisabled = false;
+}
```

Comment:
Please put them between `#ifdef USERMOD_BLE_2_JSON` ... `#endif` --> see my comment on wled.h.

Reply:
as per the convo above, i'm not sure what the community wants here... there seems to be conflicting opinions above